src/utils/generate.ts

```diff
@@ -27,12 +27,13 @@ export function generateParserUrl(
 export function generateProxyUrl(
   requestUrl: URL,
   remoteUrl: URL,
-  href: string
+  href: string,
+  subProxy?: string
 ): string {
   const realHref = getRealURL(href, remoteUrl);
 
   const urlParam = `?url=${encodeURIComponent(realHref.href)}`;
-  return `${requestUrl.origin}/proxy${urlParam}`;
+  return `${requestUrl.origin}/proxy${subProxy ? `/${subProxy}` : ''}${urlParam}`;
```

Comment:
Unreadable...  
I think we should have implemented MIME type restriction first, #98

Reply:
Maybe accept the pull request first, and then fix it under mime type restriction?